ts/components/conversation/ContactName.tsx

```diff
@@ -17,13 +18,15 @@ export const ContactName = (props: Props) => {
   const { pubkey, name, profileName, module, boldProfileName, compact, shouldShowPubkey } = props;
   const prefix = module ? module : 'module-contact-name';
 
-  const shouldShowProfile = Boolean(profileName || name);
+  const convoName = useConversationUsername(pubkey);
+
+  const shouldShowProfile = Boolean(convoName || profileName || name);
   const styles = (boldProfileName
     ? {
         fontWeight: 'bold',
       }
     : {}) as React.CSSProperties;
-  const textProfile = profileName || name || window.i18n('anonymous');
+  const textProfile = profileName || name || convoName || window.i18n('anonymous');
```

Comment:
`useConversationUsername` always returns something I think, so shouldShowProfile is always true. I don't think that is expected right?

also, about line 29, convoName, got from useConversationUserName is already `convoProps?.profileName || convoProps?.name || convoId`